debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java

```diff
@@ -2985,6 +2985,14 @@ public void parseUserDdlStatements() {
         assertThat(tables.size()).isEqualTo(0);
     }
 
+    @Test
+    @FixFor("DBZ-5836")
+    public void parseCreateUserDdlStatement() {
```

Comment:
@ani-sha Could you please move it to the sql test files for the grammar so it can be easily used for upstream PR as well?

Reply:
@jpechane As I didn't see any `CREATE USER` statements there, I thought of adding a test here. Anyways I will it in the SQL file too.

Reply:
@ani-sha ok, two places are better than one :-)